.github/ISSUE_TEMPLATE/work_item.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Work Item
```

Comment:
I guess this one is more for internal WI ?

Reply:
was wondering if we really needed, if we can incorporate it in the feature/bug or maybe rename it as infra or other

Reply:
i'm thinking of removing it. these are templates after all, we can just erase them and write what we want :)